File: dev/requirements/screen/cases/recording-indicator.2.2.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 2.2 — the red dot at the head of the header while capture runs.
let recordingIndicator_2_2 = RequirementCase(id: "2.2") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state()), slug: "recording-indicator", id: "2.2", region: .venue)
    }
}
#endif

File: dev/requirements/screen/cases/recording-indicator.2.2.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 2.2 — A red dot sits at the head of the header whenever capture is running.
final recordingIndicator_2_2 = RequirementCase('2.2', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(),
    region: StageRegion.venue,
    slug: 'recording-indicator',
    id: '2.2',
  );
});
